### DISCUSSION TITLE
secondary_items in footer

### DISCUSSION THREAD

**Discussion by azim-b** [h2oai/wave#1246] [Q&A] [answered]
Is there a reason why a `header_card()` allows `secondary_items`, so that the app can have components centred in the header; but a `footer_card()` doesn't?

With the introduction of `items` in a `footer_card()`, it has obviously become much more flexible because now you could add (right-aligned) components in the footer.

I'm just curious about the thought process / reason behind this subtle difference.

**Comment by mturoci** [comment, marked as answer]
In general, we try to keep the API surface as small as possible meaning we expose only the requested attributes that we think might be useful and reasonable. As for the footer, it currently allows for links only, but this may change in the future if a reasonable feature request comes through.
